requests/sdl.yml

```diff
@@ -0,0 +1,4 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - sdl: sdl2
+  - sdl: sdl3
```

Comment:
Do we need `sdl2-compat` here as well?

Reply:
No we do not as `sdl2-compat` upstream package is intended to be a new `sdl2` package with patch number >=50 as discussed here: https://github.com/conda-forge/staged-recipes/pull/29061#issuecomment-2653903072